src/js/components/AppPageComponent.jsx

```diff
@@ -157,7 +157,16 @@ var AppPageComponent = React.createClass({
     });
   },
 
-  onScaleAppError: function (errorMessage) {
+  onScaleAppError: function (errorMessage, statusCode, instances) {
+    if (statusCode === 409) {
+      var appId = this.state.appId;
```

Comment:
This would work with a bind
```javascript
DialogStore.handleUserResponse(dialogId, function () {
  AppsActions.scaleApp(this.state.appId, instances, true);
}.bind(this));
```
else I am also happy with
```javascript
let appId = this.state.appId;
```

@Raffo You can decide.